skiko/src/jvmMain/cpp/linux/HardwareInfo.cc

```diff
@@ -0,0 +1,58 @@
+#include <regex>
+#include <iostream>
+#include <cstdio>
+#include <memory>
+#include <stdexcept>
+#include <string>
+#include <array>
+#include "jni_helpers.h"
+
+extern "C"
+{
+    std::string process(const char* command) {
+        std::array<char, 128> buffer;
+        std::string result;
+        std::unique_ptr<FILE, decltype(&pclose)> pipe(popen(command, "r"), pclose);
+        if (!pipe) {
+            return "";
+        }
+        while (fgets(buffer.data(), buffer.size(), pipe.get()) != nullptr) {
+            result += buffer.data();
+        }
+        return result;
+    }
+
+    JNIEXPORT jstring JNICALL Java_org_jetbrains_skiko_RenderExceptionsHandlerKt_getNativeGraphicsAdapterInfo(
+        JNIEnv *env, jobject object)
+    {
+        std::stringstream stream;
+        std::string searchSentence = "VGA compatible controller: ";
+        stream << "lspci | grep " << "\"" << searchSentence << "\"";
```

Comment:
This looks somewhat hacky and shalln't belong to Skiko generic code, IMO.

Reply:
Done.